bigquery_etl/shredder/config.py

```diff
@@ -324,6 +324,20 @@ def fields(self) -> tuple[str, ...]:
         *FOCUS_ADDITIONAL_DELETIONS,
         *LEGACY_MOBILE_SOURCES,
     ),
+    DeleteTarget(
+        table="glean_telemetry_derived.cohort_weekly_active_clients_staging_v1",
+        field=(CLIENT_ID,) * 15,
+    ): (
+        DESKTOP_GLEAN_SRC,
+        DeleteSource(table="focus_android.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="firefox_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="fenix.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="klar_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="focus_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="klar_android.deletion_request", field=GLEAN_CLIENT_ID),
+        *FOCUS_ADDITIONAL_DELETIONS,
+        *LEGACY_MOBILE_SOURCES,
+    ),
```

Comment:
If this only has desktop client ids then it should only use those deletion requests
```suggestion
    client_id_target(
        table="glean_telemetry_derived.cohort_weekly_active_clients_staging_v1",
    ): DESKTOP_GLEAN_SRC,
```

Reply:
Understood, thank you!